pep-0654.rst

```diff
@@ -295,40 +295,38 @@ in the following example:
 
 .. code-block::
 
-   >>> def f(v):
-   ...     try:
-   ...         raise ValueError(v)
-   ...     except ValueError as e:
-   ...         return e
-   ...
-   >>> try:
-   ...     raise ExceptionGroup("one", [f(1)])
-   ... except ExceptionGroup as e:
-   ...     eg1 = e
-   ...
-   >>> try:
-   ...     raise ExceptionGroup("two", [f(2), eg1])
-   ... except ExceptionGroup as e:
-   ...     eg2 = e
-   ...
-   >>> import traceback
-   >>> traceback.print_exception(eg2)
-   Traceback (most recent call last):
-     File "<stdin>", line 2, in <module>
-   ExceptionGroup: two
-      ------------------------------------------------------------
-      Traceback (most recent call last):
-       File "<stdin>", line 3, in f
-      ValueError: 2
-      ------------------------------------------------------------
-      Traceback (most recent call last):
-       File "<stdin>", line 2, in <module>
-      ExceptionGroup: one
-        ------------------------------------------------------------
-        Traceback (most recent call last):
-         File "<stdin>", line 3, in f
-        ValueError: 1
-   >>>
+    >>> def f(v):
+    ...     try:
+    ...         raise ValueError(v)
+    ...     except ValueError as e:
+    ...         return e
+    ...
+    >>> try:
+    ...     raise ExceptionGroup("one", [f(1)])
+    ... except ExceptionGroup as e:
+    ...     eg1 = e
+    ...
+    >>> raise ExceptionGroup("two", [f(2), eg1])
+
+      | Traceback (most recent call last):
+      |   File "<stdin>", line 1, in <module>
+      | ExceptionGroup: two
+      |  with 2 sub-exceptions:
+      +-+---------------- 1 ----------------
+        | Traceback (most recent call last):
+        |   File "<stdin>", line 3, in f
+        | ValueError: 2
+        +---------------- 2 ----------------
+        | Traceback (most recent call last):
+        |   File "<stdin>", line 2, in <module>
+        | ExceptionGroup: one
+        |    with one sub-exception:
+        +-+---------------- 2.1 ----------------
+          | Traceback (most recent call last):
+          |   File "<stdin>", line 3, in f
+          | ValueError: 1
+        +------------------------------------
+      +------------------------------------
```

Comment:
These two "outdents" seem superfluous.

Reply:
I think there should only be one, and it should be indented with the entry immediately above it.